### PR TITLE
LLM Graph transformer dealing with empty strings

### PR DESCRIPTION
**Pull request by tomasonjo**
Pydantic allows empty strings:

```
from langchain.pydantic_v1 import Field, BaseModel

class Property(BaseModel):
  """A single property consisting of key and value"""
  key: str = Field(..., description="key")
  value: str = Field(..., description="value")

x = Property(key="", value="")
```

Which can produce errors downstream. We simply ignore those records

### CODE REVIEW

**Review by ccurme — APPROVED** [review]

File: libs/experimental/langchain_experimental/graph_transformers/llm.py [hunk 1]
```diff
@@ -576,13 +576,17 @@ def _convert_to_graph_document(
     else:  # If there are no validation errors use parsed pydantic object
         parsed_schema: _Graph = raw_schema["parsed"]
         nodes = (
-            [map_to_base_node(node) for node in parsed_schema.nodes]
+            [map_to_base_node(node) for node in parsed_schema.nodes if node.id]
```
Comment (jexp) [inline libs/experimental/langchain_experimental/graph_transformers/llm.py:579]: Should we also test for empty label?

**Review by jexp — APPROVED** [review]
Should probably check for empty label too

File: libs/experimental/langchain_experimental/graph_transformers/llm.py (hunk 1 above)
Comment (tomasonjo) [reply libs/experimental/langchain_experimental/graph_transformers/llm.py:579]: If empty label is passed it uses default "Node" label

Comment (tomasonjo) [reply libs/experimental/langchain_experimental/graph_transformers/llm.py:579]: So I wouldn't skip records if node labels are missing, as we just use default node label in that case

**Comment by tomasonjo** [comment]
Added explicit check for empty strings in node labels

**Review by jexp — APPROVED** [review]
LGTM

**Review by ccurme — APPROVED** [review]
